.github/workflows/biothings_annotator_tests.yml

```diff
@@ -0,0 +1,43 @@
+name: Biothings Annotator Tests
+
+on:
+  push: {}
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run_user_geneset_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ "3.9", "3.10", "3.11", "3.12" ]
+    steps:
+      - name: Checkout biothings_annotator source
+        uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Upgrade pip
+        run: pip install --upgrade pip
+      - name: Create Python Virtual Environment
+        run: python -m venv .venv
+      - name: Source Python Virtual Environment
+        run: source .venv/bin/activate
+      - name: Install PyTest
```

Comment:
Are we able to install our dependencies from the pyproject.toml?

Reply:
- name: Install dependencies
        run: pip install -e ".[dev]"

Reply:
yes, we are